src/routes/index.svelte

```diff
@@ -9,8 +9,7 @@
 
 			return {
 				props: {
-					games,
-					query
+					games
```

Comment:
Ik heb dit gemist denk ik. Waar is de query nu? Of is dit de homepagina?

Reply:
Als je query nodig hebt, kan je die beter uit de `store` halen, dan is die namelijk `reactive`